docs/content/library/HtmlCssSelectors.fsx

```diff
@@ -0,0 +1,308 @@
+(** 
+# F# Data: HTML CSS selectors
+
+This article demonstrates how to use HTML CSS selectors to browse the DOM of parsed HTML files.
+
+Usage of CSS selectors is a very natural way to parse HTML when we come from Web developments.
+The HTML CSS selectors are based on the [JQuery selectors](https://api.jquery.com/category/selectors/).
+*)
+
+#r "../../../bin/FSharp.Data.dll"
+open FSharp.Data
+
+(** 
+Openning FSharp.Data.CssSelectorExtensions module will enable CSS selectors.
+*)
+open FSharp.Data.CssSelectorExtensions
+
+(**
+
+## Practice 1: Search something on Google
+
+We will parse links of a Google to search for `FSharp.Data` like in `HTML Parser` article.
+*)
+let doc = HtmlDocument.Load "http://www.google.co.uk/search?q=FSharp.Data"
+
+(**
+To be sure we get search results, we will parse links in the div with id `search`.
+Then, for example, we could ensure we the HTML's structure is really compliant with the parser
+using the direct descendants selector.
+*)
+
+let links = doc.CssSelect "div#search > div#ires li.g > div.s div.kv cite"
```

Comment:
Could you change this so that the spacing looks as follows: 

    let links = 
      doc.CssSelect "div#search ..."
      |> List.map (fun n ->
          match n.InnerText() with
          | ... )

This makes the code sample a bit less wide and it will look better in the generated HTML page.